Source/ui_qt/DebugSupport/RegViewVU.cpp

```diff
@@ -0,0 +1,189 @@
+#include <stdio.h>
+#include <QHeaderView>
+#include <QMenu>
+#include "RegViewVU.h"
+#include "string_format.h"
+
+#define CLIP_FLAG_MASK 0xFFFFFF
+
+CRegViewVU::CRegViewVU(QWidget* parent, CMIPS* ctx)
+    : CRegViewPage(parent)
+    , m_ctx(ctx)
+{
+	AllocateTableEntries(2, 45 + 16);
+	setColumnWidth(0, 46);
+	horizontalHeader()->setStretchLastSection(true);
+	for(unsigned int x = 0; x < 32; x++)
+	{
+		if(x < 10)
```

Comment:
Same comment as below, there's probably a better formatting string that can be used here.